functorch/_src/aot_autograd.py

```diff
@@ -908,47 +942,44 @@ def functional_call(*args, **kwargs):
             )
         return out
 
-    def aot_function_simplified(
-        fn: Callable,
-        fw_compiler: Callable,
-        bw_compiler: Optional[Callable] = None,
-        partition_fn: Callable = default_partition,
-        decompositions: Optional[Dict] = None,
-        hasher_type=None,
-        static_argnums=None,
-    ) -> Callable:
-        assert static_argnums is None
-        if bw_compiler is None:
-            bw_compiler = fw_compiler
-        aot_config = AOTConfig(
-            fw_compiler=fw_compiler,
-            bw_compiler=bw_compiler,
-            partition_fn=partition_fn,
-            decompositions=decompositions,
-            num_params_buffers=params_len,
-        )
-
-        compiled_fn = None
+    assert static_argnums is None
+    if bw_compiler is None:
+        bw_compiler = fw_compiler
+    aot_config = AOTConfig(
+        fw_compiler=fw_compiler,
+        bw_compiler=bw_compiler,
+        partition_fn=partition_fn,
+        decompositions=decompositions,
+        num_params_buffers=params_len,
+    )
 
-        @wraps(fn)
-        def new_func(*args):
-            nonlocal compiled_fn
-            if compiled_fn is None:
-                compiled_fn = create_aot_dispatcher_function(
-                    fn,
-                    args,
-                    aot_config,
-                )
-            return compiled_fn(args)
+    joined_args = fake_flat_tensor_args + inputs
+    aot_dispatcher_function = _create_aot_dispatcher_function(functional_call, joined_args, aot_config, fake_mode)
 
-        return new_func
+    @wraps(functional_call)
+    def compiled_function(*args):
+        return aot_dispatcher_function(args)
 
-    compiled_f = aot_function_simplified(functional_call, *top_args, **top_kwargs)
+    # [Real vs Fake Params]
```

Comment:
Hey, I missed this comment in my PR, but can you please add a comment saying to move this like:

`compiled_f = aot_function_simplified(functional_call, *top_args, **top_kwargs)`

Into def `forward` as a way of disabling the cache?